exercise/migrations/0048_learningobject_consecutive_order.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.20 on 2023-07-05 07:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Any existing courses would have the consecutive_order set to 0 for all learning objects. If this field is added to the database, this should probably be fixed.